api/clients/v2/examples/client_construction.go

```diff
@@ -0,0 +1,281 @@
+package examples
+
+import (
+	"fmt"
+	"io"
+	"math/rand"
+	"path/filepath"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/api/clients/v2"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/payloaddispersal"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/payloadretrieval"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/relay"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/verification"
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/Layr-Labs/eigenda/common/geth"
+	auth "github.com/Layr-Labs/eigenda/core/auth/v2"
+	"github.com/Layr-Labs/eigenda/core/eth"
+	"github.com/Layr-Labs/eigenda/encoding"
+	"github.com/Layr-Labs/eigenda/encoding/kzg"
+	"github.com/Layr-Labs/eigenda/encoding/kzg/prover"
+	"github.com/Layr-Labs/eigenda/encoding/kzg/verifier"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+)
+
+func createPayloadDisperser(privateKey string) (*payloaddispersal.PayloadDisperser, error) {
+	logger, err := createLogger()
+	if err != nil {
+		panic(fmt.Sprintf("create logger: %v", err))
+	}
+
+	kzgProver, err := createKzgProver()
+	if err != nil {
+		return nil, fmt.Errorf("create kzg prover: %v", err)
+	}
+
+	disperserClient, err := createDisperserClient(privateKey, kzgProver)
+	if err != nil {
+		return nil, fmt.Errorf("create disperser client: %w", err)
+	}
+
+	certVerifier, err := createCertVerifier()
+	if err != nil {
+		return nil, fmt.Errorf("create cert verifier: %w", err)
+	}
+
+	payloadDisperserConfig := payloaddispersal.PayloadDisperserConfig{
+		PayloadClientConfig:    *clients.GetDefaultPayloadClientConfig(),
+		DisperseBlobTimeout:    2 * time.Minute,
+		BlobCompleteTimeout:    2 * time.Minute,
+		BlobStatusPollInterval: 1 * time.Second,
+		ContractCallTimeout:    5 * time.Second,
+	}
+
+	return payloaddispersal.NewPayloadDisperser(
+		logger,
+		payloadDisperserConfig,
+		disperserClient,
+		certVerifier,
+		nil,
+	)
+}
+
+func createRelayPayloadRetriever() (*payloadretrieval.RelayPayloadRetriever, error) {
+	logger, err := createLogger()
+	if err != nil {
+		return nil, fmt.Errorf("create logger: %w", err)
+	}
+
+	// we don't need signing functionality for this eth client
+	ethClient, err := createEthClient(logger)
```

Comment:
what does this mean? I don't see anything related to signing on this constructor. is there another constructor that has signer? comment is a bit confusing imo

Reply:
Whoops, vestigial comment from some functionality that got simplified. Removed [9de543b2](https://github.com/Layr-Labs/eigenda/pull/1454/commits/9de543b29d26c8a87d399114b207ad65d54df797)